edx_proctoring/tests/utils.py

```diff
@@ -306,7 +306,7 @@ def _create_onboarding_attempt(self):
         """
         Create onboarding attempt
         """
-        return self._create_exam_attempt(self.onboarding_exam_id)
+        return self._create_exam_attempt(self.onboarding_exam_id, is_practice_exam=True)
```

Comment:
I think this was an oversight, so I marked that onboarding exams should also be considered practice exams